### DISCUSSION TITLE
Regions and Districts taxonomies are registered as public

### DISCUSSION THREAD

**Discussion by mdavoli** [code4recovery/12-step-meeting-list#963] [General]
### Contact Details

no response

### Website With Issue

https://staging.orlandorecovery.com/recovery-blog/tsml_region/altamonte-springs/

### What happened?

Currently the region and disctrict custom taxonomies are public so if view one of the taxonomies you get a broken archive page for them. I don’t believe this is intended and it should probably be disabled to prevent indexing accidentally via search engines.

https://developer.wordpress.org/reference/functions/register_taxonomy/#:~:text=taxonomy%20is%20for.-,public,-bool

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

### What browsers are you seeing the problem on?

Chrome

**Comment by tim-rohrer** [comment]
Please provide a screen cap of what is broken. Someone smarter than me might already understand this, and be have to validate it, but give the fact that this hasn’t come up before…

Thanks!

**Reply by mdavoli** [reply]
Broken, isn't the word I'd use. More so it's not ideal for SEO to have public pages for unformatted content. Since these taxonomies are not intended to be front-facing by themselves and instead are used to power the TSML-UI filtering leaving them public is unnecessary. Of course, @97eighty  is correct, alternatively you could build in redirects so anyone attempt to visit those URLS would instead be send to the correct URL within the app.

**Comment by 97eighty** [comment]
We can hide them from the front end as @mdavoli pointed out. The flip side is we support them and rewrite the URL to open TSML-UI with the appropriate filters pre-selected i.e. ?region=altamonte-springs
